kobo/apps/project_views/views.py

```diff
@@ -56,6 +57,7 @@ def get_queryset(self, *args, **kwargs):
     def assets(self, request, uid):
         if not user_has_view_perms(request.user, uid):
             raise Http404
+        self._paginator = FastAssetPagination()
```

Comment:
I think we can specify the paginator class in the arguments passed to `@action` decorator. 


```python 
    @action(
        detail=True,
        methods=['GET'],
        filter_backends=[SearchFilter, AssetOrderingFilter],
        pagination_class=FastAssetPagination
    )
    def assets(self, request, uid):
        if not user_has_view_perms(request.user, uid):
```